### DISCUSSION TITLE
File Not Found. Error: Unable to access jarfile C:\Windows\apktool_0.jar

### DISCUSSION THREAD

**Discussion by Eldonz0** [iBotPeaches/Apktool#2804] [General]
CMD :

C:\Windows>apktool
File Not Found
Error: Unable to access jarfile C:\Windows\apktool_0.jar

Can anyone please help me with this I already walked through the process exactly as it should be, what am I missing?

**Comment by iBotPeaches** [comment]
It shouldn't be looking for `apktool_0.jar` so my guess is a step was missed somewhere.
